gui/api/resources.py

```diff
@@ -958,15 +960,30 @@ def lock(self, request, **kwargs):
 
         bundle, obj = self._get_parent(request, kwargs)
 
-        if obj.vol_encrypt == 0:
-            raise ImmediateHttpResponse(
-                response=self.error_response(request, _('Volume is not encrypted.'))
-            )
```

Comment:
Why was this removed?

Reply:
So we already raise a validation error when locking a pool if that pool is not encrypted. This was redundant in that case